### ISSUE TITLE
Remove internal node backend code in Specter 2.0

### ISSUE THREAD

**Issue by moneymanolis** [cryptoadvance/specter-desktop#2093]

**Comment by AtmegaBuzz** [comment]
@moneymanolis  do we have to remove whole InternalNode class implementation and its methods that are called in other files

**Comment by moneymanolis** [comment]
Good, question, I'd say yes. Double-checking with @k9ert - do you agree?

**Comment by k9ert** [comment]
> @moneymanolis do we have to remove whole InternalNode class implementation 
> and its methods that are called in other files

Yes, i would say so. If someone wants to preserve the Internal Node feature, i'd be happy if that code goes over to a new (external-) extension. Shouldn't be too difficult as we already have a spectrum extension which created an own node. Some links:
* [Intro in extension development](https://docs.specter.solutions/desktop/extensions/intro/) (video: https://www.youtube.com/watch?v=s41RFyRn1I8)
* Specifics about adding nodes: https://docs.specter.solutions/desktop/extensions/nodes/
* the [spectrum-extension](https://github.com/cryptoadvance/specter-desktop/tree/master/src/cryptoadvance/specterext/spectrum) and how it solved the issue of a new node created in an extension
